packages/fetchai/connections/p2p_libp2p/dht/dhtnode/message.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package dhtnode;
+
+message AgentRecord {
```

Comment:
`AgentRecord` is the PoR

Reply:
You already named it, but wouldn't it have been better to name it something like `AgentPoR`...I am not suggesting you do it now, this is ok as is.